src/main/firebase/FirebaseController.ts

```
@@ -369,6 +369,17 @@ export class FirebaseDatabase {
         });
         return messages;
     }
+
+    async createVirtualDevice(virtualDevice: any, userId: string) {
+        const virtualdeviceCreated = await this.db.ref()
```

you can just return await, but the await is only necessary if you want to throw the exception here, if not, just return